src/mono/mono/eventpipe/ds-rt-mono.h

```diff
@@ -0,0 +1,138 @@
+// Implementation of ds-rt.h targeting Mono runtime.
+#ifndef __DIAGNOSTICS_RT_MONO_H__
+#define __DIAGNOSTICS_RT_MONO_H__
+
+#include <config.h>
+
+#ifdef ENABLE_PERFTRACING
+#include "ep-rt-mono.h"
+
+#undef DS_LOG_ALWAYS_0
+#define DS_LOG_ALWAYS_0(msg)
```

Comment:
why not `g_warning` or `mono_trace`?

Reply:
Was planning to wire these up in a later PR as well as some additional logging feature currently used in coreclr implementation.